client/src/App.js

```diff
@@ -9,14 +10,17 @@ const Container = styled.div`
 `;
 
 const App = () => (
-  <Container>
-    <header>
-      <h1>shorty</h1>
-    </header>
-    <main>
-      <Shorties />
-    </main>
-  </Container>
+  <>
+    <GlobalStyle />
```

Comment:
You could move `<GlobalStyle />` to the `index.js` as it contains styles that are targeted at things outside of your app (e.g. `body`).

Reply:
I prefer to add `GlobalStyle` to `App`, because it might depend on it. For example, you like to render the App on server side. Loading the `index.js` on server side is difficult, because you usually use it to subscribe to other "applications" like the service workers of progressive web apps.
In my understanding, the App is the whole React application with all dependencies required to render it. 
The `index.js` is used to run other Apps too.

I understand your point, but the app usually targets many things outside the container where it's rendered. If you take a look at [react-helmet](https://github.com/nfl/react-helmet) or update the HTML`<title>`  from your app.

Reply:
Got it, then leave it as it is.